test/unit/model/relation/task/ItemToMediaRelationMigrationTaskTest.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoMediaManager\test\unit\model\relation\task;
+
+use common_exception_MissingParameter;
+use oat\generis\test\TestCase;
+use oat\tao\model\task\migration\service\MigrationConfigFactory;
+use oat\tao\model\task\migration\service\QueueMigrationService;
+use oat\tao\model\task\migration\MigrationConfig;
+use oat\tao\model\task\migration\service\ResultFilter;
+use oat\tao\model\task\migration\service\ResultFilterFactory;
+use oat\tao\model\task\migration\service\SpawnMigrationConfigService;
+use oat\tao\model\taskQueue\QueueDispatcherInterface;
+use oat\tao\model\taskQueue\Task\CallbackTaskInterface;
+use oat\taoMediaManager\model\relation\service\ItemToMediaRdsSearcher;
+use oat\taoMediaManager\model\relation\task\ItemToMediaRelationMigrationTask;
+use oat\taoMediaManager\model\relation\task\ItemToMediaUnitProcessor;
+use PHPUnit\Framework\MockObject\MockObject;
+
+class ItemToMediaRelationMigrationTaskTest extends TestCase
+{
+    private const CHUNKSIZE_EXAMPLE = 1;
+    private const START_EXAMPLE = 0;
+    private const PICKSIZE_EXAMPLE = 2;
+    private const REPEAT_EXAMPLE = true;
+
+    /** @var ItemToMediaUnitProcessor|MockObject */
+    private $processorMock;
+
+    /** @var ItemToMediaRelationMigrationTask */
+    private $subject;
+
+    /** @var QueueMigrationService|MockObject */
+    private $queueMigrationServiceMock;
+
+    /** @var QueueDispatcherInterface|MockObject */
+    private $queueDispatcherInterfaceMock;
+
+    /** @var ItemToMediaRdsSearcher|MockObject */
+    private $itemToMediaRdsSearcher;
+    /** @var SpawnMigrationConfigService|MockObject */
+    private $spawnMigrationConfigServiceMock;
+    /** @var ResultFilterFactory|MockObject */
+    private $resultFilterFactoryMock;
+    /** @var MigrationConfigFactory|MockObject */
+    private $migrationConfigFactoryMock;
```

Comment:
```suggestion
    /** @var ItemToMediaRdsSearcher|MockObject */
    private $itemToMediaRdsSearcher;
    
    /** @var SpawnMigrationConfigService|MockObject */
    private $spawnMigrationConfigServiceMock;
    
    /** @var ResultFilterFactory|MockObject */
    private $resultFilterFactoryMock;
    
    /** @var MigrationConfigFactory|MockObject */
    private $migrationConfigFactoryMock;
```
New lines missing to be consistent with previous ones